examples/hybrid-module/package.json

```diff
@@ -49,12 +49,11 @@
     "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-prettier": "^4.0.0",
     "jest": "^27",
-    "jest-junit": "^13",
-    "jsii": "^1.55.1",
+    "jest-junit": "^15",
+    "jsii": "1.x",
     "jsii-diff": "^1.55.1",
     "jsii-docgen": "^6.2.4",
     "jsii-pacmak": "^1.55.1",
-    "json-schema": "^0.4.0",
     "npm-check-updates": "^16",
     "prettier": "^2.6.2",
     "projen": "^0.53.8",
```

Comment:
For consistency's sake I feel like we should update these as well, this is a pretty old version.